setup.py

```diff
@@ -45,7 +45,7 @@
     extras_require={
         "dev": ["black", "isort", "pytest<8.0", "flake8", "pre-commit"],
         "atari": ["gym[atari, accept-rom-license]"],
-        "mujoco": ["gym[mujoco]", "mujoco", "mujoco_py>=2.0,<3.0"],
+        "mujoco": ["gym[mujoco]"],
```

Comment:
Removing mujoco_py fixes the error -> only used for deprecated mujoco versions no longer supported by gym